opm/parser/eclipse/EclipseState/Tables/PlymaxTable.hpp

```diff
@@ -34,11 +34,14 @@ namespace Opm {
          * \brief Read the PLYMAX keyword and provide some convenience
          *        methods for it.
          */
-        void init(Opm::DeckItemConstPtr item)
+        void init(Opm::DeckRecordConstPtr record)
         {
-            SimpleTable::init(item,
-                              std::vector<std::string>{"C_POLYMER", "C_POLYMER_MAX"});
-
+            createColumns(std::vector<std::string>{"C_POLYMER", "C_POLYMER_MAX"});
```

Comment:
Thanks for testing - I'll merge as is now; but I am the process of reworking these things anyway. Will come back to your suggestion.